File: src/Around.Core/Entities/Rent.cs
using System;

namespace Around.Core.Entities
{
    public class Rent
    {
        public int Id { get; set; }

        public int ClientId { get; set; }

        public int CopterId { get; set; }

        public DateTime StartTime { get; set; }

        public virtual Client Client { get; set; }

        public virtual Copter Copter { get; set; }

        public virtual Cheque Cheque { get; set; }

        public Rent CreateFromDto(RentAggregate rentDto)
        {
            ClientId = rentDto.ClientId;
            CopterId = rentDto.CopterId;
            StartTime = DateTime.Now;

            return this;
        }
    }
}

File: src/Around.Frontend/Around-Frontend/src/app/model/rent.ts
import { RentDto } from '../interface/rent-dto';

export class Rent {
    constructor(
        public id: number,
        public clientId: number,
        public copterId: number,
        public startTime: Date) {
    }
    public static Create(dto: RentDto): Rent {
        return new Rent(dto.id, dto.clientId, dto.copterId,
             dto.startTime);
    }
}
